bundle/language_en.properties

```
@@ -794,8 +794,10 @@ angal.medicalstock.chooseavalidmovementdate.msg
 angal.medicalstock.chooseavalidpreparationdate                                                         = Choose a valid preparation date
 angal.medicalstock.clickdrugs                                                                          = Double click to show lot details
 angal.medicalstock.cost.col                                                                            = Cost
+angal.medicalstock.deletemovementsuccess.msg                                                           = Last movement successfully deleted.
 angal.medicalstock.discharge.btn                                                                       = Discharge
 angal.medicalstock.discharge.btn.key                                                                   = D
+angal.medicalstock.deletelastmovementdenied.msg                                                        = Denied to delete the last movement.
```

Please update core accordingly:

```suggestion
angal.medicalstock.notpossibletodeletethismovementthemedicalhasbeenusedafterbeenreceivedinward.fmt.msg                                                        = Not possible to delete this movement: the medical {0} has already been used after been received in ward {1}. 
```